src/main/java/rtg/world/WorldTypeRTG.java

```diff
@@ -80,10 +53,30 @@ public IChunkProvider getChunkGenerator(World world, String generatorOptions)
             return new ChunkProviderGenerate(world, world.getSeed(), world.getWorldInfo().isMapFeaturesEnabled());
         }
     }
-	
+
     @Override
-    public float getCloudHeight()
-    {
+    public float getCloudHeight() {
         return 256F;
     }
+
+    private static Runnable clearChunkManager() {
+        return new Runnable() {
+            public void run() {
+                chunkManager = null;
+            }
+        };
+    }
+
+    private static Runnable clearChunkProvider() {
+        return new Runnable() {
+            public void run() {
+                chunkProvider = null;
+            }
+        };
+    }
+
+    public boolean isWorldTypeRTG() {
```

Comment:
The Runnable methods? I never added them, they were [added by @Zeno410](https://github.com/Team-RTG/Realistic-Terrain-Generation/commit/7704310c85590959fb4eef6879c19a879e66e92f), like 2 months ago. I just moved the methods in the file. (Well, automatic code clean-up did.)

The Runnable methods don't do anything until the server is stopped, at which time they reset the RTG Chunk Manager/Provider.

I did, however, add isWorldTypeRTG(). What's wrong with that?